scene/resources/curve.cpp

```diff
@@ -1438,15 +1440,16 @@ void Curve3D::_bake() const {
 
 	Vector3 position = points[0].position;
 	real_t dist = 0.0;
-	List<Plane> pointlist;
+	List<Plane> pointlist; // abuse Plane for (position, dist)
```

Comment:
```suggestion
	List<Plane> pointlist; // Abuse Plane for (position, dist).
```
Vector 4 is a standard data type too.

Reply:
I changed the comment. But I think there is no point to switch to Vector4, since it is another "abuse", and is even less convenient to retrieve the position(first 3 components).